### DISCUSSION TITLE
Facilities data

### DISCUSSION THREAD

**Discussion by dougrichardson** [opennem/opennem#364] [Q&A]
From the OpenNEM website, I can't find where the information for generation facilities is pulled from. It is hard to tell if it is updated regularly, or is a realistic representation of real current facilities. I am especially interested in the low number of committed wind facilities on OpenNEM compared to e.g. this Ecogeneration map: https://www.ecogeneration.com.au/wp-content/uploads/2023/03/2023_WindMap.pdf

Could someone please advise on the provenance of the facilities data?

**Comment by aleith** [comment]
in my experience it's from various sources. and gets updated when people notice inconsistencies. having tried to reconcile differences in sources for facility level info, be it nameplate capacities of generation units, commissioning, decommissioning dates and what have you, there's a lot of different info around about some facilities, and even naming can be confusing in locations with a lot of generation, e.g. Kalgoorlie OCGTs :-) 
One source is the classification UUID system that SCADA data from AEMO uses.
Another is the Fed Govt Office of Greenhouse Gas Regulation safeguard mechanism data.
Wikipedia, and a bunch of new webbed data portals looking to commercialise publicly available info that is behind various APIs making it difficult for non-programmers to access the data.
